lib/Parse/Lexer.cpp

```diff
@@ -1373,6 +1373,64 @@ void Lexer::lexEscapedIdentifier() {
   formToken(tok::backtick, Quote);
 }
 
+/// Find the end of a version control conflict marker.
+static const char *findConflictEnd(const char *CurPtr, const char *BufferEnd,
+                                   ConflictMarkerKind CMK) {
+  const char *terminator = CMK == ConflictMarkerKind::Perforce ? "<<<<\n"
+                                                               : ">>>>>>>";
+  size_t termLen = CMK == ConflictMarkerKind::Perforce ? 5 : 7;
```

Comment:
Nitpick: I think it would be better to use strlen here, or even better just use a StringRef from the start.